src/Features/Core/Portable/Formatting/IEditorFormattingService.cs

```diff
@@ -8,7 +8,7 @@
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Text;
 
-namespace Microsoft.CodeAnalysis.Editor
+namespace Microsoft.CodeAnalysis
 {
     internal interface IEditorFormattingService : ILanguageService
```

Comment:
1. if we're going to move this, i would say that it shoudl be modernized.  so things like IList should be immtuable array
2. really not sure about things like SUpprotsFormatOnPaste.  Things like Paste are a very editor oriented topic.
3. This woudl affects TS and F# i believe as they use these.
4. Not sure whyt his needs to move to FEatures layer... what's the scenario that needs this that doesn't have access to the editor layer?

Reply:
> 4. Not sure whyt his needs to move to FEatures layer... what's the scenario that needs this that doesn't have access to the editor layer?

You're probably expecting this from me at this point, but: LSP and/or Omnisharp. See https://github.com/dotnet/roslyn/issues/50221 and https://github.com/dotnet/roslyn/issues/50129#issuecomment-751148095.